src/main/java/io/pinecone/PineconeIndexOperationClient.java

```diff
@@ -102,8 +102,9 @@ public void configureIndex(String indexName, ConfigureIndexRequest configureInde
     }
 
     private Request buildRequest(String method, String path, String acceptHeader, RequestBody requestBody) {
+        String completeUrl = (path.equals(EMPTY_RESOURCE_PATH)) ? url : url + "/" + path;
```

Comment:
Trailing slash was causing the issue for create and listIndexes for gcp-starter env. I have ran integration test suite for both pod and gcp-starter to make sure it works.